docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -233,32 +237,32 @@ Try `tsh login` with a local user. Use a custom `KUBECONFIG` to prevent overwrit
 the default one in case there is a problem.
 
 ```bash
-$ KUBECONFIG=${HOME?}/teleport.yaml tsh login --proxy=tele.example.com:443 --user=alice
+KUBECONFIG=${HOME?}/teleport.yaml tsh login --proxy=tele.example.com:443 --user=alice
 ```
 
-Teleport updated `KUBECONFIG` with a short-lived 12 hour certificate.
+Teleport updated `KUBECONFIG` with a short-lived 12-hour certificate.
 
 ```bash
 # List connected Kubernetes clusters
-$ tsh kube ls
+tsh kube ls
 
-Kube Cluster Name Selected
------------------ --------
-tele.example.com
+# Kube Cluster Name Selected
+# ----------------- --------
+# tele.example.com
 
 # Login to Kubernetes by name
-$ tsh kube login tele.example.com
-Kube Cluster Name Selected
------------------ --------
-tele.example.com  *
+tsh kube login tele.example.com
+# Kube Cluster Name Selected
+# ----------------- --------
+# tele.example.com  *
 
 # Once working, remove the KUBECONFIG= override to switch to teleport
-$ KUBECONFIG=${HOME?}/teleport.yaml kubectl get -n teleport-cluster pods
-NAME                                READY   STATUS    RESTARTS   AGE
-teleport-cluster-6c9b88fd8f-glmhf   1/1     Running   0          127m
+KUBECONFIG=${HOME?}/teleport.yaml kubectl get -n teleport-cluster pods
+# NAME                                READY   STATUS    RESTARTS   AGE
```

Comment:
I've left these commented out since the tab contains a single code block with multiple commands and outputs.